Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -153,6 +161,11 @@ export function runQTest(args: QTestArguments): Result {
             "--qTestRawArgFile ",
             Artifact.input(args.qTestRawArgFile)
         ),
+        Cmd.option(
+            "--qCodeCoverageEnumType ",
+            qCodeCoverageEnumTypeToString(args.qCodeCoverageEnumType)

```

Comment:
If we use the args to switch on/off this feature, we will need to announce it to Office and they need to change all their build definitions. For BuildXL selfhost build we need to update qtestframework.dsc to pass args to enable the feature. I would prefer to just turn it on in this SDK.

Reply:
Added code to pass the arguments in qtestframework.dsc